### PR TITLE
Update wgpu to 22.1.0

### PR DESCRIPTION
**Pull request by jerzywilczek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 ### 💥 Breaking changes
 
+- The `LIVE_COMPOSITOR_REQUIRED_WGPU_FEATURES` environment variable no longer accepts the value `SHADER_UNUSED_VERTEX_OUTPUT`, since this feature was removed in wgpu.
+- wgpu was updated from 0.19 to 22.0. All breaking changes in wgpu APIs that are exposed in the compositor apply here too.
```
Comment (WojciechBarczynski) [inline CHANGELOG.md:8]:
```suggestion
- wgpu was updated from 0.19 to 22.0. All breaking changes in wgpu APIs exposed in the LiveCompositor apply here too.
```

File: compositor_render/src/transformations/text_renderer.rs [hunk 2]
```diff
@@ -206,13 +209,15 @@ impl From<&TextComponent> for TextParams {
...
+    cache: Mutex<Cache>,
 }
 
 impl TextRendererCtx {
-    pub(crate) fn new() -> Self {
+    pub(crate) fn new(device: &wgpu::Device) -> Self {
         Self {
             font_system: Mutex::new(FontSystem::new()),
             swash_cache: Mutex::new(SwashCache::new()),
+            cache: Mutex::new(Cache::new(device)),
```
Comment (WojciechBarczynski) [inline compositor_render/src/transformations/text_renderer.rs:220]: If those values are always used together, use one struct and one mutex and lock on it.

Comment (jerzywilczek) [reply compositor_render/src/transformations/text_renderer.rs:220]: theyre not always used together

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 ### 💥 Breaking changes
 
+- The `LIVE_COMPOSITOR_REQUIRED_WGPU_FEATURES` environment variable no longer accepts the value `SHADER_UNUSED_VERTEX_OUTPUT`, since this feature was removed in wgpu.
+- wgpu was updated from 0.19 to 22.0. All breaking changes in wgpu APIs that are exposed in the LiveCompositor apply here too.
```
Comment (wkozyra95) [inline CHANGELOG.md:8]: - Keep the changelog format and language style as below
- Update link in https://compositor.live/docs/deployment/configuration#live_compositor_required_wgpu_features
- I would prefer to keep this section only for stuff that would force use to release a new major version. Given how often wgpu releases new major I'm not sure whether we should consider this a breaking change.

I would either:

### Option 1

if there are significant changes that are user facing
```suggestion
- Update `wgpu`  from 0.19 to 22.0. ([#733](https://github.com/software-mansion/live-compositor/pull/733)) by [@jerzywilczek](https://github.com/jerzywilczek)
```

### Option 2

or if you think that supported feature change is only/main change
```suggestion
- Drop support for `SHADER_UNUSED_VERTEX_OUTPUT` `wgpu` feature.  ([#733](https://github.com/software-mansion/live-compositor/pull/733)) by [@jerzywilczek](https://github.com/jerzywilczek)
```
and in the others section add
```
- Update `wgpu`  from 0.19 to 22.0. ([#733](https://github.com/software-mansion/live-compositor/pull/733)) by [@jerzywilczek](https://github.com/jerzywilczek)
```

### Option 3

Do not introduce breaking change, keep the env working just disable behavior for that env

Comment (WojciechBarczynski) [reply CHANGELOG.md:8]: option 3 doesn't make sense IMO

**Review by WojciechBarczynski — APPROVED** [review]

**Pull request merged**